### PR TITLE
GLTFLoader: Move isNotColorData property to ITextureInfo

### PR DESCRIPTION
**Pull request by Popov72**

### CODE REVIEW

**Review by deltakosh — APPROVED** [review]

File: loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts [hunk 1]
```diff
@@ -19,6 +19,7 @@ export class KHR_texture_basisu implements IGLTFLoaderExtension {
     public enabled: boolean;
 
     private _loader: GLTFLoader;
+    private _textureInfo: ITextureInfo;
```
Comment (bghgary) [inline loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts:22]: You can also try storing textureInfo on the ITexture interface. Then you won't need this anymore.

File: loaders/src/glTF/2.0/glTFLoaderInterfaces.ts [hunk 2]
```diff
@@ -217,6 +217,8 @@ export interface ITexture extends GLTF2.ITexture, IArrayItem {
  * Loader interface with additional members.
  */
 export interface ITextureInfo extends GLTF2.ITextureInfo {
+    /** false or undefined if the texture holds color data (true if data are roughness, normal, ...) */
+    isNotColorData?: boolean;
```
Comment (bghgary) [inline loaders/src/glTF/2.0/glTFLoaderInterfaces.ts:221]:
```suggestion
    isNonColorData?: boolean;
```
I would also drop the `is` part of it, but that's not a big deal.

**Review by bghgary — COMMENTED** [review]
Small suggestions, but looks good otherwise.

File: loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts [hunk 3]
```diff
@@ -34,7 +33,8 @@ export class KHR_texture_basisu implements IGLTFLoaderExtension {
 
     /** @hidden */
     public loadTextureInfoAsync(context: string, textureInfo: ITextureInfo, assign: (babylonTexture: BaseTexture) => void): Nullable<Promise<BaseTexture>> {
-        this._textureInfo = textureInfo; // need to save this for use in _loadTextureAsync below
+        const texture = ArrayItem.Get(`${context}/index`, this._loader.gltf.textures, textureInfo.index);
+        texture._textureInfo = textureInfo; // need to save this for use in _loadTextureAsync below
```
Comment (bghgary) [inline loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts:37]: You probably want to do this in the core loader and not just for this extension.

Comment (Popov72) [reply loaders/src/glTF/2.0/Extensions/KHR_texture_basisu.ts]: True, corrected!

**Review by bghgary — APPROVED** [review]

**Pull request merged**